examples/ice-single-port/index.html

```diff
@@ -0,0 +1,52 @@
+<html>
+  <head>
+    <title>ice-restart</title>
```

Comment:
Probably a copy-paste mistake. Needs to be change to `ice-single-port`

Reply:
Fixed! nice catch :)